src/util/edit.cc

```diff
@@ -70,7 +70,7 @@ void EditEngine::modifyBytes(size_t pos, const data::BinData& bytes,
   if (next_it == address_mapping_.cend() || end_pos <= next_it.key()) {
     // This is the case when whole query range is located in only one node.
     if (it->fragment_ == nullptr) {
-      // TODO(catsuryuu): code improvement
+      // TODO(catsuryuu): little refactoring
```

Comment:
Still too generic. Please write exactly what's wrong or what should be improved here, otherwise no one other than you can fix this ;)